dropbox/dropbox.py

```diff
@@ -600,6 +685,8 @@ def _get_dropbox_client_with_select_header(self, select_header_name, team_member
         new_headers[select_header_name] = team_member_id
         return Dropbox(
             self._oauth2_access_token,
+            self._oauth2_refresh_token,
+            self._oauth2_access_token_expiration,
```

Comment:
I think refresh token makes sense, as it is the same idea as the long lived access token.  I can see the benefit of only sharing refresh and not sharing access tokens but think that is a nit

Reply:
Yeah the only concern for existing code is. If the SDK copies 10 clients. It will end up with refreshing 10 clients at the same time. You might wanna introduce some randomness to the buffer time then?